DeviceCode/Targets/Native/STM32F4/DeviceCode/STM32F4_GPIO/STM32F4_GPIO_functions.cpp

```diff
@@ -37,25 +35,28 @@ typedef struct {
     BYTE expected; // expected pin state
     GPIO_INTERRUPT_SERVICE_ROUTINE ISR; // interrupt handler
     void* param;   // interrupt handler parameter
-} STM32F4_Int_State;
+    UINT32 debounceTicks;
+};
 
-static STM32F4_Int_State g_int_state[STM32F4_Gpio_MaxInt]; // interrupt state
+static STM32F4_Int_State g_int_state[ STM32F4_Gpio_MaxInt ]; // interrupt state
 
 static UINT32 g_debounceTicks;
-static UINT16 g_pinReserved[TOTAL_GPIO_PORT]; //  1 bit per pin
-
+static UINT16 g_pinReserved[ TOTAL_GPIO_PORT ]; //  1 bit per pin
 
 /*
  * Debounce Completion Handler
  */
-void STM32F4_GPIO_DebounceHandler (void* arg)
+void STM32F4_GPIO_DebounceHandler( void* arg )

```

Comment:
It was done seperately as a distinct commit as per recommendations, However given the poor quality diff and review system avaialable on GitHub that recommendation may need to be revised...